about.html

```diff
@@ -0,0 +1,152 @@
+      <img
+        id="nav-btn"
+        class="nav-btn hide-desktop"
+        src="./assets/icons/menu-btn.svg"
+        alt="menu"
+      />
+      <h3 class="about-hello poppins hide-desktop">"Hello AI innovators!"</h3>
+      <h1 class="about-title lato">
+        Exploring the Frontiers of Intellingent Technologies
+      </h1>
+
+      <p class="about-description lato">
+        Welcome to the International Conference on Artificial Intelligence
+        (ICAI), the premier event for researchers, practitioners, and
+        enthusiasts in the field of AI. This year's conference will feature
+        keynote speeches from some of the top experts in the industry, as well
+        as technical sessions, workshops, and networking opportunities.
+      </p>
+
+      <p class="contact-text lato">
+        Please contact us per Email if you have any questions about
+        ArtificialEdge conference!
+      </p>
+      <p class="email lato">support@artificialedge.com</p>
+    </section>
+
+    <section class="logo-section">
+      <h2 class="section-title lato">Artificial Edge Conference Logo</h2>
+      <div class="orange-divisor"></div>
+      <p class="logo-description lato">
+        The logo design for the conference was chosen to be simple and
+        minimalist, relying on the inclusion of the company name to convey
+        confidence and authority in the field of artificial intelligence. This
+        clean design can be effective in terms of the audience's ability to
+        remember the company name and associate it with the conference theme.
+      </p>
+      <div class="img-container">
+        <img
+          class="logo-img"
+          src="./assets/img/logo.png"
+          alt="image of the logo"
+        />
+      </div>
+    </section>
+    <section class="history-section">
+      <h2 class="section-title lato">
+        See the past ArtificialEdge Conferences
+      </h2>
+      <div class="orange-divisor"></div>
+      <p class="logo-description lato">
+        Take a look at the past ArtificialEdge Conference wich took place in New
+        York City, and Tokyo.
+      </p>
+      <div class="history-container">
+        <div class="history-card nyork">
+          <h3 class="card-year lato">2019</h3>
+          <p class="card-desc lato">
+            ArtificialEdge Conference in New York, USA.
+          </p>
+        </div>
+        <div class="history-card tokyo">
+          <h3 class="card-year lato">2021</h3>
+          <p class="card-desc lato">
+            ArtificialEdge Conference in Tokyo, Japan.
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <footer class="footer">
+      <img
+        class="logo-footer"
+        src="./assets/img/logo-blanco.png"
+        alt="logo image"
+      />
+      <p class="footer-text lato">
+        © 2023 Artificial Edge. <br />
+        Some Rights Reserved.
+      </p>
+    </footer>
+    <script src="about.js"></script>
+  </body>
+</html>
```

Comment:
- [ ] The mobile view of your About Page is missing the Partner's section based on the design template provided for this project. It would be nice to include it in your design for the About Page. 👍 😎 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/233149379-5b04b0f3-80ca-4f8a-b2c5-7a16ba04f112.png)

###  Behance Design

![image](https://user-images.githubusercontent.com/77106962/233149193-74d6dbfa-5eaf-4e74-8582-c748a7a8586b.png)